### PR TITLE
Fix async main and JSON indentation in sample

### PR DESCRIPTION
**Pull request by snobu**

### CODE REVIEW

File: src/Cnab.JsonConverters/ParameterConverter.cs [hunk 1]
```diff
@@ -44,6 +44,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
         private Parameter populateParameter(JsonReader reader, JsonSerializer serializer, Parameter destination)
         {
             serializer.Populate(reader, destination);
+
```
Comment (radu-matei) [inline src/Cnab.JsonConverters/ParameterConverter.cs]: Ugh, I clearly miss `gofmt` and `golint`...

File: src/Cnab.Bundle/Bundle.cs [hunk 2]
```diff
@@ -33,11 +34,12 @@ public class Bundle
         [JsonProperty("actions")]
         public Dictionary<string, Action> Actions { get; set; }
 
-        public static Bundle LoadUnsigned(string fileName)
+        public static async Task<Bundle> LoadUnsignedAsync(string fileName)
```
Comment (radu-matei) [inline src/Cnab.Bundle/Bundle.cs]: Do we want to have both sync and async methods?
What is the best practice in this case?

**Comment by radu-matei** [comment]
This looks great, thanks for your contribution!

While we're discussing the sync/async API, could you please, also sign your commits?

Thanks!

File: src/Cnab.Bundle/Bundle.cs (hunk 2 above)
Comment (snobu) [reply src/Cnab.Bundle/Bundle.cs]: Async methods everywhere, unless there's very clear reasoning to have it sync, or the super edge case where you want both.
Ref: https://blogs.msdn.microsoft.com/pfxteam/2012/04/13/should-i-expose-synchronous-wrappers-for-asynchronous-methods/

I'll fix the README as well, missed that.

Comment (radu-matei) [reply src/Cnab.Bundle/Bundle.cs]: The slight difference here is that we would not return `.Result` from an async method, but rely on `StreamReader` and `JsonConvert` to do their operations synchronously.

That being said, I'm having an asynchronous API for now and come back later and see if a synchronous one is actually needed.
cc @itowlson, who I think had his fair share of `sync` vs. `async` kerfuffles.

Comment (snobu) [reply src/Cnab.Bundle/Bundle.cs]: 10 Points for `kerfuffles`.

File: src/Cnab.JsonConverters/ParameterConverter.cs (hunk 1 above)
Comment (snobu) [reply src/Cnab.JsonConverters/ParameterConverter.cs]: OmniSharp or [this one](https://marketplace.visualstudio.com/items?itemName=Leopotam.csharpfixformat). I don't really trust formatters, apologies.

**Comment by snobu** [comment]
Commit signed.

**Pull request merged**